Dockerfile-Postgres

```diff
@@ -0,0 +1,64 @@
+##################################################
+# BUILDER CONTAINER
+##################################################
+
+FROM openjdk:11.0.12 as builder
+
+USER root
+
+WORKDIR /usr/src/dependencies
+
+# INSTALL MAKE
+RUN apt update \
+    && apt install build-essential -y
+
+# INSTALL POSTGRESQL
+RUN wget https://ftp.postgresql.org/pub/source/v14.0/postgresql-14.0.tar.gz \
+    && tar -zxf postgresql-14.0.tar.gz \
+    && cd postgresql-14.0 \
+    && ./configure --without-readline --without-zlib\
+    && make \
+    && make install \
+    && export PATH="/usr/local/psql:${PATH}"
```

Comment:
@alextsaihi when I tried this I had to use `export PATH="/usr/local/pgsql:${PATH}"` (pgsql and not psql) and it was able to recognize psql command after setting this PATH variable. I would check if that's the case here as well.

Reply:
@alextsaihi 
Also, if you're not already familiar, both the Dockerfile-postgres and Dockerfile consists of three stages:

Stage 1: focuses on installing Postgres (SQLite in the case of Dockerfile) and building the database.

Stage 2: JAR file is generated.

Stage 3: Executing the application using the JAR file.

It's important to note that in stage 3, the database file (created in stage 1) is copied and used. However, note that the PostgreSQL executable isn't accessible within this stage. Hence, attempting to run PostgreSQL commands within stage 3 (and even stage 2) won't function **as per the design**. However PostgreSQL commands do work within stage 1 where you need it to create and load the db (I can confirm this based on my own testing).

Reply:
Thanks for the explanation, this does help clarify the process of Dockerfiles and Makefiles for me. 

However, the issue that I was having was from the line in Dockerfile that calls for psql refresh, which runs the psql commands from the Makefile that does db building and populating. When it tries to run this step, it tells me that psql is not found.